sdk/servicebus/Azure.Messaging.ServiceBus/stress/.gitignore

```diff
@@ -0,0 +1,12 @@
+stress-test-resources.json
```

Comment:
Should this be removed?

Reply:
Some of the other languages have it to keep some of the files generated when deploying the tests out of the repo but I don't think its necessary